### PR TITLE
fix：组件sql语句字段校验、提示优化

### PR DESCRIPTION
**Pull request by ghost**
English | [简体中文](https://github.com/opentiny/tiny-engine/blob/develop/.github/PULL_REQUEST_TEMPLATE/PULL_REQUEST_TEMPLATE.zh-CN.md)

# PR

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our [Commit Message Guidelines](https://github.com/opentiny/tiny-engine/blob/develop/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] Built its own designer, fully self-validated

## PR Type

What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

### What is the current behavior?

Issue Number: #273 

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: scripts/connection.mjs [hunk 1]
```diff
@@ -94,12 +94,37 @@ class MysqlConnection {
     return str.replace(/'/g, "\\'")
   }
 
+  /**
+   * 校验组件字段数据
+   * @param {object} component 组件数据
+   * @returns boolean 校验组件字段是否失败，true-有字段出错
+   */
+  validateFields(component) {
```
Comment (gene9831) [inline scripts/connection.mjs]: 函数名 `validateFields` 不太直观，返回布尔型的函数名应该为 `isXXX`

File: scripts/connection.mjs [hunk 2]
```diff
@@ -94,12 +94,37 @@ class MysqlConnection {
...
+    })
+  }
+
   /**
    * 生成更新组件的sql语句
    * @param {object} component 组件数据
    * @returns 更新组件的sql语句
    */
   updateComponent(component) {
+    const valid = this.validateFields(component)
```
Comment (gene9831) [inline scripts/connection.mjs]: `valid` 的字面含义和 `validateFields` 函数返回值的含义是相反的。还有 L192 行

File: scripts/connection.mjs (hunk 1 above)
Comment (ghost) [reply scripts/connection.mjs]: 已修改

File: scripts/connection.mjs (hunk 2 above)
Comment (ghost) [reply scripts/connection.mjs]: 已修改

**Review by chilingling — APPROVED** [review]

**Review by kedenLi — APPROVED** [review]

**Pull request merged**